File: JPAFoodFinder/src/main/java/com/skilldistillery/foodfinder/entities/Address.java
package com.skilldistillery.foodfinder.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Address {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String street_address;
	private String city;
	private String state;
	private String zip;
	// map service_location 
	// map to donor 
	// map to recipient 

}

File: ngFoodfinder/src/app/models/address.ts
export class Address {

  id : number;
  streetAddress : string;
  city : string;
  state : string;
  zip : string;

  constructor(
    id : number = 0,
    streetAddress : string = '',
    city : string = '',
    state : string = '',
    zip : string = ''
  ) {
    this.id = id;
    this.streetAddress = streetAddress;
    this.city = city;
    this.state = state;
    this.zip = zip;
  }

}
